src/main/java/SaveFileManager.java

```diff
@@ -0,0 +1,70 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SaveFileManager {
+	private static final String PATH_TO_SAVE_FILE = "./data/duke.txt";
+	private static final String SAVE_FILE_DIRECTORY= "./data";
+	private static final int MAX_TASKS = 100;
+
+	public static ArrayList<Task> readFromSaveFile() {
+		ArrayList<Task> tasks = new ArrayList<>(MAX_TASKS);
+		int indexOfTask = 0;
+		String taskType;
+		int isDone;
```

Comment:
Try to avoid naming int variables like a boolean. If the variable is indeed to be used as a boolean variable then i suggest changing the type to boolean. However if it is meant to be an int then maybe could use some other naming